tools/github/setup.sh

```diff
@@ -20,6 +20,8 @@
 #then echo skipping setup ; exit 0
 #fi
 
+set -e
```

Comment:
It is required to finish on an error.
Now it works as expected.
<img width="1329" alt="image" src="https://github.com/apache/openwhisk/assets/3447251/a3db7a6b-bf1c-4577-ab72-4bfb2fea45a3">

I will revert the temporary commit.